boards/shields/st_b_cams_omv_mb1683/st_b_cams_omv_mb1683.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include <zephyr/dt-bindings/video/video-interfaces.h>
```

Comment:
you don't need to include `video-interfaces.h` here as it does not use anything in this file (?)

Reply:
Removed. Thank you, @ngphibang!